client/src/components/Form/Elements/FormNumber.vue

```diff
@@ -34,12 +34,12 @@ export default {
             validator: (prop) => ["integer", "float"].includes(prop.toLowerCase()),
         },
         min: {
-            type: Number,
+            type: [Number, String],
```

Comment:
Why did you do this ? That smells like we're hiding a problem here.

Reply:
Yes, correct. The form dictionaries from the visualization serialization provide these values as strings not as numeric values, leading to console prop mismatch errors. This fix resolves this issue but the number component has to be revisited to properly show the alert for values which are not in the min/max interval.